public/app/features/explore/Logs/Logs.tsx

```diff
@@ -152,6 +153,8 @@ class UnthemedLogs extends PureComponent<Props, State> {
   cancelFlippingTimer?: number;
   topLogsRef = createRef<HTMLDivElement>();
   logsVolumeEventBus: EventBus;
+  static contextType = ContentOutlineContext;
```

Comment:
is this used?

Reply:
yes, it's a way for class-based components to get access to Context without Context.Consumer component. otherwise it throws an error during render like this:
<img width="408" alt="Screenshot 2024-04-18 at 2 32 27 PM" src="https://github.com/grafana/grafana/assets/58232930/e3c1a85c-7cd7-4686-afd9-67a1cce8114e">